uaclient/tests/test_sso.py

```diff
@@ -85,6 +85,24 @@ def __str__(self):
         return self.exception_string
 
 
+class TestSSOAuthError:
+
+    @pytest.mark.parametrize(
+        'error_key,value',
+        ((sso.API_ERROR_2FA_REQUIRED, '2FA required'),
+         (sso.API_ERROR_INVALID_CREDENTIALS, 'Invalid pwd')))
+    def test_api_errors_as_dict_keyed_on_error_code(self, error_key, value):
+        api_errors = [{'code': sso.API_ERROR_2FA_REQUIRED,
+                       'message': '2FA required'},
+                      {'code': sso.API_ERROR_INVALID_CREDENTIALS,
+                       'message': 'Invalid pwd'}]
+        exception = SimpleSSOAuthError("Invalid 2fa", api_errors=api_errors)
+        if value is None:
```

Comment:
`value` is never None.

Reply:
I forgot to add the third parameterized tuple. just added it.